client/app/scripts/components/node-details/node-details-table.js

```diff
@@ -109,7 +109,7 @@ function getSortedNodes(nodes, sortedByHeader, sortedDesc) {
   const getValue = getValueForSortedBy(sortedByHeader);
   const withAndWithoutValues = groupBy(nodes, (n) => {
     const v = getValue(n);
-    return v !== null && v !== undefined ? 'withValues' : 'withoutValues';
+    return !n.valueEmpty && v !== null && v !== undefined ? 'withValues' : 'withoutValues';
```

Comment:
Living dangerously checking props of `n` here first, before checking `null`-ness.